scripts/generate-artifacts.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+# Please, run this script from the root folder!
+
+if [ -z "$1" ]; then
+    echo "Error: Please provide the name of the package"
+    exit 1
+fi
+
+PACKAGE=$1
+ARTIFACTS_PATH=packages/$PACKAGE/artifacts
+CIRCUIT_PATH=packages/$PACKAGE/circuits
+PTAU=13
+
+if [ "$2" ]; then
+    PTAU=$2
```

Comment:
add a log here that explains whether or not we're defaulting to a particular ptau file